src/Simulation/Simulators.Tests/CoreTests.cs

```diff
@@ -43,6 +43,30 @@ public void BasicExecution()
             Assert.Empty(error.ToString().Trim());
         }
 
+        [Fact]
+        public void ClassicControlSimulatorSupport()
```

Comment:
Also a minor thing: Everywhere else the code talks about ClassicalControl opposed to ClassicControl, so I'd suggest to adapt the name for both tests to match that.

Reply:
done